cloudbuild.yaml

```diff
@@ -1,24 +1,20 @@
 timeout: 1800s
-steps:  
+steps:
 - name: gcr.io/cloud-builders/bazel
   args: ['run', '//base:static']
 - name: gcr.io/cloud-builders/docker
   args: ['tag', 'bazel/base:static', 'gcr.io/$PROJECT_ID/static:latest']
 
 - name: gcr.io/cloud-builders/bazel
```

Comment:
I wonder we can just remove `run //base:base` too. Unsure. For `run //cc:cc` too.